compose-prod.yml

```diff
@@ -1,20 +1,25 @@
 api:
    build: ./
    command: nodemon api/api.js
-   links:
-      - db
-      - broker
-   ports:
-      - "3900:3900"
+   net: host
```

Comment:
Is that the part where the network interface of the Docker is mapped directly to the host?